.github/workflows/flaky.yaml

```diff
@@ -55,12 +55,36 @@ jobs:
           result=$(echo '${{ toJSON(needs) }}' | jq -r .tests.result)
           echo TESTS_RESULT=$result
           echo "TESTS_RESULT=$result" >>"$GITHUB_ENV"
+      - name: download nextest reports
+        uses: actions/download-artifact@v4
+        with:
+          pattern: libtest_run_${{ github.run_number }}-${{ github.run_attempt }}-*
+          merge-multiple: true
+          path: nextest-results
+      - name: create summary report
+        id: make_summary
+        run: |
+          shopt -s nullglob
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
```

Comment:
why not pick something fixed that's not valid json?  in theory this could produce valid json i guess?

Reply:
this is just a string of characters, I commented the line since getting a new one every time seems unnecessary, but the line tells us how we got it. The syntax 

```
{name}<<{delimiter}
{value}
{delimiter}
```

source: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

Reply:
ah, whoops.  made the same comment twice :)